backend/src/main/java/site/coduo/pairroommember/service/dto/PairRoomMemberResponse.java

```diff
@@ -0,0 +1,17 @@
+package site.coduo.pairroommember.service.dto;
+
+import site.coduo.pairroom.repository.PairRoomEntity;
+
+public record PairRoomMemberResponse(Long id,
+                                     String status,
+                                     String navigator,
+                                     String driver,
+                                     String accessCode
+) {
```

Comment:
```suggestion
public record PairRoomMemberResponse(
        Long id,
        String status,
        String navigator,
        String driver,
        String accessCode
) {
```
id도 들여쓰기를 해주는 것은 어떨까요?!
문서화도  추가해 주세요!